scripts/ipfs/install

```diff
@@ -5,7 +5,10 @@ set -e
 GO_IPFS_VERSION=v0.4.6
 
 BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-
+# Check for PI user if missing add it
+if [ -z $(cat /etc/passwd | grep pi:x) ];then
+    useradd pi
```

Comment:
Can't we just use `~` to get the existing home directory instead of making a new user? This IPFS installer wouldn't be the appropriate place to make a new user anyway.